### ISSUE TITLE
WrongAnnotationError lintFix problem

### ISSUE THREAD

**Issue by tasomaniac** [vanniktech/lint-rules#283]
The code I had has `JvmStatic` and `Provides` in wrong order. But while running `lintFix`, I had the following error. 

I'm on Android Tools 3.3

```
  Caused by: java.lang.IllegalArgumentException: Did not find "@Provides @JvmStatic @NotNull" in "@Provides
      @JvmStatic
      internal fun provideRequests(retrofit: Retrofit) = retrofit.create<AuthenticationRequests>()" as suggested in the quickfix. Consider calling ReplaceStringBuilder#range() to set a larger range to search than the default highlight range.
        at com.android.tools.lint.LintFixPerformer.addReplaceString(LintFixPerformer.kt:493)
        at com.android.tools.lint.LintFixPerformer.addEdits(LintFixPerformer.kt:244)
        at com.android.tools.lint.LintFixPerformer.registerFix(LintFixPerformer.kt:94)
        at com.android.tools.lint.LintFixPerformer.findApplicableFixes(LintFixPerformer.kt:209)
        at com.android.tools.lint.LintFixPerformer.fix(LintFixPerformer.kt:100)
        at com.android.tools.lint.LintCliClient.run(LintCliClient.java:258)
        at com.android.tools.lint.gradle.LintGradleClient.run(LintGradleClient.java:243)
```

**Comment by vanniktech** [comment]
Oh yes. That's probably kotlin code and Kotlin adds the `NotNull` annotation that is not present in the source code.

**Comment by vanniktech** [comment]
Fixed by #290
